.github/workflows/publish_pure_python.yml

```diff
@@ -91,6 +91,7 @@ jobs:
         with:
           test_extras: ${{ inputs.test_extras }}
           test_command: ${{ inputs.test_command }}
+          python-version: ${{ inputs.python-version }}
```

Comment:
Does this not need a default specified somewhere to maintain backward compatibility?

Reply:
Yes. I think the default value is '3.x'. I didn't implement that here because I wasn't sure how the templating logic worked.

Reply:
Ok I *think* I've now implemented the argument correctly and have just used the same defaults as the `build-python-dist` workflow.